web/ocr-fileformat.js

```diff
@@ -1,4 +1,6 @@
 /* globals $ */
+/* globals Blob */
+/* global Prism */
```

Comment:
Is there a difference between `globals` (as in line 2) and `global` (as in line 3) here?

Reply:
No.